filters/apiusagemonitoring/spec.go

```diff
@@ -169,6 +183,23 @@ func (s *apiUsageMonitoringSpec) buildUnknownPathInfo(paths []*pathInfo) *pathIn
 	return s.unknownPath
 }
 
+func (s *apiUsageMonitoringSpec) configInfo(apis []*apiConfig) apiConfigContext {
+	info := apiConfigContext{}
+
+	for _, api := range apis {
+		context := api.Context
+		if context.ConfigID == "" || context.ConfigCreated.IsZero() {
```

Comment:
Context is optional. This code doesn't look to support this case. (nil reference on `context.ConfigID`)

Reply:
it is guaranteed. The `apiConfig.Context` field is not a pointer.